api/controllers/node.js

```diff
@@ -152,9 +152,9 @@ NodeController.updateForgingStatus = function(context, next) {
 		return next(new Error('Access Denied'));
 	}
 
-	var publicKey = context.request.swagger.params.data.value.publicKey;
-	var password = context.request.swagger.params.data.value.password;
-	var forging = context.request.swagger.params.data.value.forging;
+	const publicKey = context.request.swagger.params.data.value.publicKey;
+	const password = context.request.swagger.params.data.value.password;
+	const forging = context.request.swagger.params.data.value.forging;
 
```

Comment:
@diego-G @MaciejBaj I don't like to suggest stuff outside of the scope of the PR but changing 3 lines above to: 
```suggestion
const { publicKey, password, forging } = context.request.swagger.params.data.value;
```
would be nicer.

Reply:
Let's ignore the comment above. I think we will fix those when we implement `prefer-destructuring`